packages/dds/tree/src/feature-libraries/editable-tree/fence.json

```diff
@@ -2,5 +2,5 @@
     "tags": [ "forest-proxy" ],
     "exports": ["index"],
     // TODO: "feature-libraries" dependency is for defaultFieldKinds, and should be removed once proper view schema are provided.
-    "imports": ["schema-stored","tree","dependency-tracking","forest","util","feature-libraries","modular-schema"]
+    "imports": ["schema-stored","tree","dependency-tracking","forest","util","feature-libraries","modular-schema","domains","checkout","shared-tree"]
```

Comment:
"domains" should not be depended on here. It is for code that is specific to a particular domain (ex: the Json Domain).

General code what works with all domains should not do there: typedJsonCursor should prabably be moved to feature-libraries instead.